GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -1875,6 +1875,32 @@ public void loadTexture(TextureCallback callback,
     }
 
     /**
+     * Loads atlas information file placed in the assets folder.
+     *
+     * Atlas information file contains in UV space the informations of offset and
+     * scale for each mesh mapped in some atlas texture.
+     * The content of the file is at json format like:
+     *
+     * [ {name: SUN, offset.x: 0.9, offset.y: 0.9, scale.x: 0.5, scale.y: 0.5},
+     * {name: EARTH, offset.x: 0.5, offset.y: 0.9, scale.x: 0.5, scale.y: 0.5} ]
+     *
+     * @param resource
+     *            A stream containing a text file on JSON format.
+     *
+     * @return List of atlas informations load.
+     */
+    public List<GVRAtlasInformation> loadTextureAtlasInformations(GVRAndroidResource resource) {
+
+        assertGLThread();
```

Comment:
@danke-sra, ok.
You are right, I will wait for more comments of review and fix all on next update.